basics/basics-final/lib/posts.js

```diff
@@ -1,8 +1,9 @@
 import fs from 'fs';
 import path from 'path';
 import matter from 'gray-matter';
-import { remark } from 'remark';
-import html from 'remark-html';
+import {unified} from 'unified';
+import remarkParse from 'remark-parse'
```

Comment:
This comment is more for the repository maintainers than the submitter.

I understand these two dependencies are included in the [remark dependencies](https://github.com/remarkjs/remark/blob/5017a27db024db6feec85a3e1e19f8d78a485680/packages/remark/package.json#L39-L44) and should be available when installing the remark package as instructed elsewhere in the guide.

I'm commenting to mention that it would be helpful for the https://nextjs.org/learn/basics/dynamic-routes/render-markdown page to include `npm install unified remark-parse remark-html` instead of `npm install remark remark-html` so that the imports used in this file are explicit dependencies.

At first glance, this repository seems to be example code for reference from the guide, and not the code for the actual guide, but I have not gone down the rabbit hole trying to find the actual content for the guide.